lib/uncruft/deprecation_handler.rb

```diff
@@ -64,7 +66,7 @@ def remove_line_number(message)
     end
 
     def gem_home(message)
-      message.match(%r{called from( .+ at)? (#{ENV['GEM_HOME']}/(.+/)*gems)})&.[](2)
+      message.match(%r{(?i:c)alled from( .+ at)? (#{ENV['GEM_HOME']}/(.+/)*gems)})&.[](2) # rubocop:disable Style/FetchEnvVar
```

Comment:
```suggestion
      message.match(%r{(?i:c)alled from( .+ at)? (#{ENV.fetch('GEM_HOME', nil)}/(.+/)*gems)})&.[](2)
```

I'm not sure if the code is intended to break if GEM_HOME isn't present, but at the very least we could `.fetch` with a nil default.